### DISCUSSION TITLE
Please help me with login.

### DISCUSSION THREAD

**Discussion by AdvitSingh5** [vercel/vercel#9612] [Help] [answered]
I want to deploy my project. 
Please help me deploy it. 
I am having issues with login.
I am trying with Github login, it asks for phone no. when i give it, it says no. is associated with a vercel account.
ok.
but i try with the normal mail. 
and it again asks for the same thing. 
then it says try again later after 1 min 
why? 
please help.

**Comment by AdvitSingh5** [comment]
damn it.. anyone answer

**Comment by leerob** [comment, marked as answer]
Hey there, apologies for this. For login issues, please contact Vercel support 🙏 

https://vercel.com/help#contact
